src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+
+const sortList = (listOfValue) => {
+  const listWithNumber = [...listOfValue];
+
+  function normalizeNumber() {
+    for (const item of listWithNumber) {
+      item.dataset.salary = item.dataset.salary.split(',').join('').slice(1);
+    }
+  };
+
+  normalizeNumber();
+
+  let sort = listWithNumber.sort((a, b) => b.dataset.salary - a.dataset.salary);
+
+  sort = sort.forEach(item => list.append(item));
+
+  return sort;
+};
+
+function getEmployees(listOfValue) {
+  const result = [...listOfValue].map(item => ({
```

Comment:
map doesn't mutate original array and returns a new one anyway, so you don't need to copy listOfValues here